Formula/perl.rb

```diff
@@ -43,6 +30,10 @@ def install
       end
     end
 
+    # We don't want the build process to go snooping around externally
+    # for things like the `Test` perl module.
+    ENV.delete("PERL5LIB")
```

Comment:
I mean, it literally broke my local build 3 times before I worked out what was going on.